src/components/HTMLEngineProvider/HTMLRenderers/ConciergeLinkRenderer.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import type {StyleProp, TextStyle} from 'react-native';
+import type {CustomRendererProps, TPhrasing, TText} from 'react-native-render-html';
+import {TNodeChildrenRenderer} from 'react-native-render-html';
+import * as HTMLEngineUtils from '@components/HTMLEngineProvider/htmlEngineUtils';
+import Text from '@components/Text';
+import useThemeStyles from '@hooks/useThemeStyles';
+import {navigateToConciergeChat} from '@userActions/Report';
+
+type ConciergeLinkRendererProps = CustomRendererProps<TText | TPhrasing>;
+
+function ConciergeLinkRenderer({tnode}: ConciergeLinkRendererProps) {
+    const styles = useThemeStyles();
+
+    // Define link style based on context
+    let linkStyle: StyleProp<TextStyle> = styles.link;
+
+    // Special handling for links in RBR to maintain consistent font size
+    if (HTMLEngineUtils.isChildOfRBR(tnode)) {
+        linkStyle = [
+            styles.link,
+            {
+                fontSize: HTMLEngineUtils.getFontSizeOfRBRChild(tnode),
+            },
+        ];
+    }
+
+    return (
+        <Text
+            style={linkStyle}
+            onPress={() => navigateToConciergeChat()}
```

Comment:
stable ref as per [this comment](https://github.com/Expensify/App/pull/67725#discussion_r2252880246) plz